File: demo/src/main/java/com/foodtruck/demo/services/LunchService.java
package com.foodtruck.demo.services;

import com.foodtruck.demo.dto.LunchDto;
import com.foodtruck.demo.models.Lunch;

public interface LunchService {

    Lunch findById(Long id);

    Iterable<Lunch> getAll();

    Lunch save(Lunch lunch);

    void update(Long id, LunchDto lunch);

    void delete(Long id);
}

File: api/src/main/kotlin/com/foodtruck/api/services/LunchService.kt
package com.foodtruck.api.services

import com.foodtruck.api.models.Lunch

interface LunchService {
    fun findById(id: Long): Lunch
    fun getAll(): Iterable<Lunch>
    fun save(lunch: Lunch): Iterable<Lunch>
    fun update(id: Long, lunch: Lunch): Iterable<Lunch>
    fun delete(id: Long)
}